src/lib.rs

```diff
@@ -381,25 +482,20 @@ impl WasccHost {
     fn add_actor_gantry_first(&self, actor: &str) -> Result<()> {
         if actor.len() == 56 && actor.starts_with('M') {
             // This is an actor's public subject
-            self.add_actor(Actor::from_gantry(&actor)?)
+            self.add_actor_from_gantry(actor)
         } else {
             self.add_actor(Actor::from_file(&actor)?)
         }
     }
 
-    #[cfg(not(feature = "gantry"))]
```

Comment:
`add_actor_from_path` is still called from `apply_manifest` if I compile with just ` features = ["manifest"]` (i.e. no `gantry` feature).